guides/common/modules/con_troubleshooting-dnf-modules.adoc

```diff
@@ -0,0 +1,8 @@
+[id="Troubleshooting_DNF_Modules_{context}"]
```

Comment:
This file is not included anywhere, so the "Troubleshooting DNF Modules" section is gone from Foreman and Katello guides now.

Reply:
Yes. We will be including that file in this PR #1694. Right now we are just baking it ready to get included, according to Ewoud's [comment](https://github.com/theforeman/foreman-documentation/pull/1694#pullrequestreview-1177344280).

Reply:
No, what @evgeni is saying is something that I already mentioned in https://github.com/theforeman/foreman-documentation/pull/1832#discussion_r1029274010. In https://github.com/theforeman/foreman-documentation/pull/1832#issuecomment-1320079343 you can see that the sections get dropped from the manual. This is still blocking the merge.

Reply:
Addressed :+1: